test/runtests.jl

```diff
@@ -67,14 +67,6 @@ init_time = time()
     println()
 
     include("quickstart.jl")
-
-    @testset "Test Benchmark script" begin
-        empty!(ARGS)
-        push!(ARGS, "KrylovBICGSTAB")
-        push!(ARGS, "CPU")
-        push!(ARGS, joinpath(INSTANCES_DIR, "case300.m"))
-        include(joinpath(BENCHMARK_DIR, "benchmarks.jl"))
-    end
```

Comment:
Shouldn't we test the benchmark code? Just with a small example...

Reply:
no you are right, we should test the benchmark code indeed